public_suffix_list.dat

```diff
@@ -12216,6 +12218,7 @@ us.reclaim.cloud
 ch.trendhosting.cloud
 de.trendhosting.cloud
 jele.club
+amscompute.com
```

Comment:
domain expires in 10 months
no _psl txt record to validate

Reply:
@dnsguru  please accept my apologies, my colleague  accidentally merged new domain to a wrong branch in our PSL fork,
asked him to revert changes and arrange another pull request for new domain once everything will be ready for that
thank you